src/data/exchange.graphql

```diff
@@ -211,11 +248,47 @@ type RejectOrderPayload {
   order: Order
 }
 
+# Autogenerated input type of SetPayment
+input SetPaymentInput {
```

Comment:
These seem tangential to what the PR is doing?

Reply:
this is auto-generated latest Exchange GraphQL schema copied over here, will follow up with PRs for adding setPayment and setShipping.

Reply:
a-ha! Makes sense.